apm-lambda-extension/extension/http_server_test.go

```diff
@@ -168,6 +169,24 @@ func Test_handleInfoRequest(t *testing.T) {
 	}
 }
 
+type errReader int
+
+func (errReader) Read(p []byte) (n int, err error) {
```

Comment:
nit, if you're not using any of the variables you could change the method sig to
```golang
func (errReader) Read(_ []byte) (int, error)
```